stripe/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -334,11 +334,13 @@ internal class PaymentSheetViewModel internal constructor(
                 confirmPaymentSelection(paymentSelection)
             }
             is StripeGooglePayContract.Result.Error -> {
+                logger.error("Error processing Google Pay payment", googlePayResult.exception)
                 eventReporter.onPaymentFailure(PaymentSelection.GooglePay)
                 paymentIntent.value?.let { it ->
                     resetViewState(
                         it,
                         googlePayResult.googlePayStatus?.getErrorResourceID()
+                            ?: R.string.stripe_google_pay_error_internal
```

Comment:
StripeGooglePayContract has a function called getErrorResourceID.  Can we move this logic into that function?

Reply:
Yes, just need a small change since this is for the case when `googlePayStatus` is null.